modules/tensor_mechanics/src/materials/LAROMANCEPartitionStressUpdateBase.C

```diff
@@ -17,14 +17,21 @@ LAROMANCEPartitionStressUpdateBaseTempl<is_ad>::validParams()
 {
   InputParameters params = LAROMANCEStressUpdateBaseTempl<is_ad>::validParams();
   params.addClassDescription("LAROMANCE base class for partitioned reduced order models");
+  params.addRangeCheckedParam<Real>(
+      "finite_difference_width",
+      1.0e-4,
```

Comment:
@aruybalid This had a huge impact with convergence on the 5 element problem you sent me. Can you help me determine a good value here? Maybe @arovinelli has some input too since he did the initial implementation. Note, that this is a fractional not absolute term now.

Reply:
Alright, My own analysis has showed:
| FD width | Wall time | Num NL |
| --- | --- | --- |
| 1e-5 | 161 | 2846 |
| 1e-4 | 133 | 2786 |
| 1e-3 | 96 | 2769 |
| 1e-2 | 84 | 2769 |
| 5e-1 | 84 | 2820 |
| 1e-1 | 80 | 2636 |
| 1 | 90 | 2821 |

So I think I'll change the default to `1.0e-2`

Reply:
I did another check with trying out doing the finite difference the other direction. It didn't help much, so I'll stick with 1e-2.

| FD width | Wall time | Num NL |
| --- | --- | --- |
| 1e-4 | 73 | 1306 |
| 1e-3 | 65 | 1317 |
| 1e-2 | 58 | 1283 |
| 1e-1 | 60 | 1355 |
| -1e-1 | 60 | 1403 |
| -1e-2 | 58  | 1288 |
| -1e-3 | 75 | 1367 |

Reply:
@tophmatthews sorry for the late reply, these have been busy days
The threshold is relative to what it is relative to what? the initial value? if so what happens if the initial value is zero? To say the truth, I'm always hesitant to use relative thresholds for a material solve. Have you  made sure the absolute residual value at convergence is close to the approximate zero we are willing to accept?